invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -34,6 +34,10 @@ li ~ .item{
   padding-left: 0em !important;
 }
 
+.right.floated.right.aligned.column {
```

Comment:
quick comment: just wondering why this (and the other override) is not needed on the oauthclient PR.

Reply:
Hmm in the `invenio-oauth2server` there are no css rules defined, as I understand we define them in `invenio-theme` and we get them from here